Makefile

```diff
@@ -0,0 +1,121 @@
+DEFAULT_GOAL := help
+
+# Ensure everything works even if GOPATH is not set, which is often the case.
+# The `go env GOPATH` will work for all cases for Go 1.8+.
+GOPATH            ?= $(shell go env GOPATH)
+
+TMP_GOPATH        ?= /tmp/thanos-go
+GOBIN             ?= $(firstword $(subst :, ,${GOPATH}))/bin
+GOPROXY           ?= https://proxy.golang.org
+GIT               ?= $(shell which git)
+
+# Version from 12.12.2019
+GOJSONTOYAML_VERSION    ?= bf2969bbd742d117a9524b859fb417fefb67565d
+GOJSONTOYAML            ?= $(GOBIN)/gojsontoyaml-$(GOJSONTOYAML_VERSION)
+
+# Version from 9.02.2020. We use https://github.com/google/go-jsonnet for ease of building.
+GO_JSONNET_VERSION         ?= 31b9ace0f65a5a5717ed9521b57d8d9e18f24070
+GO_JSONNET                 ?= $(GOBIN)/jsonnet-$(GO_JSONNET_VERSION)
+# v0.2.0
+JSONNET_BUNDLER_VERSION ?= 184841238bb5df1b886ccaecd3e179bc55c17687
+JSONNET_BUNDLER         ?= $(GOBIN)/jb-$(JSONNET_BUNDLER_VERSION)
```

Comment:
Why not pin the actual release instead of the sha?

Reply:
Just how the script that is proven for Thanos takes commit.